deploy/olm-catalog/ibm-platform-api-operator/3.6.0/ibm-platform-api-operator.v3.6.0.clusterserviceversion.yaml

```diff
@@ -299,7 +300,7 @@ spec:
                   value: quay.io/opencloudio/icp-audit-service@sha256:24a220374c39652f8b031637d852995bfaec5be2601f9e4c41eb1925531f466d
                 - name: OPERAND_PLATFORM_API_IMAGE
                   value: quay.io/opencloudio/icp-platform-api@sha256:e6a535b448608f6ca0bb9a8b3a81159174f4c4042e15257abf529289afa1ae0d
-                image: quay.io/opencloudio/ibm-platform-api-operator:latest
+                image: quay.io/opencloudio/ibm-platform-api-operator:sha256:6d39ea1f5b8956bfcef7cdc0aa087427c5fa501ea5735a22f6531c4045a6b2e3
```

Comment:
use @sha256?

Reply:
Thanks. it's resolved in https://github.com/IBM/ibm-platform-api-operator/pull/61